README.md

```diff
@@ -1,50 +1,78 @@
-
-## Contents
+### Prerequisites
+- A GitHub account
+- [Node.js and Git](https://nodejs.org/)
+- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=academic-0000-jopapa) installed
+- The [SWA CLI](https://www.npmjs.com/package/@azure/static-web-apps-cli) installed
+- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=academic-0000-jopapa) installed
 
-The apps written in the following JavaScript frameworks/libraries:
+See instructions to install, setup and run the application locally in the **README.md** file located at the root of the respective folders.
 
 | Folder          | Description                                                                          | Live Demo                                                |
 | --------------- | ------------------------------------------------------------------------------------ | -------------------------------------------------------- |
 | **angular-app** | [Sample Angular app](https://github.com/johnpapa/shopathome/blob/master/angular-app) | [angular.shopathome.dev](https://angular.shopathome.dev) |
-| **api**         | [Sample Azure Functions app](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
+| **api**         | [Fastify API](https://github.com/johnpapa/shopathome/blob/master/api) | protected                                                |
 | **react-app**   | [Sample React app](https://github.com/johnpapa/shopathome/blob/master/react-app)     | [react.shopathome.dev](https://react.shopathome.dev)     |
 | **svelte-app**  | [Sample Svelte app](https://github.com/johnpapa/shopathome/blob/master/svelte-app)   | [svelte.shopathome.dev](https://svelte.shopathome.dev)   |
 | **vue-app**     | [Sample Vue app](https://github.com/johnpapa/shopathome/blob/master/vue-app)         | [vue.shopathome.dev](https://vue.shopathome.dev)         |
 
-## Prerequisites
+## Run Shop at Home on Azure 
+> [!NOTE]
+> Ensure you fork the project to your GitHub account before you start deploying.
 
-- A GitHub account
-- [Node.js and Git](https://nodejs.org/)
-- [Visual Studio Code](https://code.visualstudio.com/?WT.mc_id=academic-0000-jopapa) installed
-- The [Static Web Apps extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurestaticwebapps&WT.mc_id=shopathome-github-jopapa&WT.mc_id=academic-0000-jopapa) installed
-- The [SWA CLI](https://www.npmjs.com/package/@azure/static-web-apps-cli) installed
-- The [Azure Functions Core Tools](https://docs.microsoft.com/azure/azure-functions/functions-run-local?WT.mc_id=academic-0000-jopapa) installed
+[![Deploy to Azure button](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#view/Microsoft_Azure_PaasServerless/StarshotTemplateGallery.ReactView)

```

Comment:
The link should be below:

https://portal.azure.com/#view/Microsoft_Azure_PaasServerless/StarshotHome.ReactView

Please look at my PR here to update the steps - https://github.com/johnpapa/shopathome/pull/191/files

Reply:
Updated accordingly - thanks!